packages/node_modules/@webex/plugin-meetings/src/meeting/util.js

```diff
@@ -91,6 +91,14 @@ MeetingUtil.isPinOrGuest = (err) => {
   return false;
 };
 
+MeetingUtil.isGuest = (locus) => {
```

Comment:
@chrisadubois i think setLocus parses all the values and sets in the meeting object so we dont need this function